wordle-app/pages/index.vue

```diff
@@ -0,0 +1,87 @@
+<template>
+  <v-container>
+
+      <!-- Single row to display multiple blocks of game pages -->
+      <v-row class="text-center pa-5">
+        <!-- Word of the Day Game Block -->
+        <v-col cols="12" md="6" class="mb-4">
+          <v-card 
+          class="game-card pa-5 d-flex flex-column align-center justify-center" 
+          outlined 
+          @click="router.push('/WordOfTheDayGame')"
+          @mousedown="pressEffect"
+          @mouseup="releaseEffect"
+          @mouseleave="releaseEffect"
+          elevation="2" 
+          hover>
+            <v-icon size="x-large">mdi-calendar-today</v-icon>
```

Comment:
Cool icons! If you wanted you could maybe make these cards their own components.